package.json

```diff
@@ -12,16 +12,13 @@
     "test:browser": "aegir test -t browser",
     "release": "aegir release",
     "release-minor": "aegir release --type minor",
-    "release-major": "aegir release --type major",
-    "coverage": "aegir coverage",
-    "coverage-publish": "aegir coverage publish"
+    "release-major": "aegir release --type major"
```

Comment:
Even though aegir doesn't support it I've kept a `coverage` script as `nyc --reporter=text --reporter=lcov npm run test:node` because `npm run coverage` is way easier to remember.